Makefile

```diff
@@ -77,9 +77,7 @@ db/setup-test: db/up db/create-user db/create-test db/migrate-test
 # start only database containers
 .PHONY: db/up
 db/up:
-	@mkdir -p docker_data && docker-compose up -d postgres
-	@until docker exec $(pg_docker_image) pg_isready; do echo 'Waiting Postgres...' && sleep 1; done
-	@sleep 2
+	make compose-up container=postgres
```

Comment:
I don't think this approach will work if you need to wait for the dependency to be ready. The end of `db/up` target is not a guarantee that Postgres is accepting connections on the container.